tools/run_tests/python_utils/upload_rbe_results.py

```diff
@@ -18,7 +18,7 @@
 import os
 import json
 import sys
-import urllib2
+import urllib.request, urllib.error, urllib.parse
```

Comment:
let's double check the upload_rbe_results.py script actually still works -  if not, we'd silently looks the bigquery data for go/grpc-flaky and we really don't want that.